utils/device.py

```diff
@@ -0,0 +1,24 @@
+import torch
```

Comment:
All python code should be under `DeOldify`, in this case `from deoldify import utils`.